src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -154,13 +162,18 @@ uct_ib_mlx5_devx_reg_ksm_data_mt(uct_ib_mlx5_md_t *md, int atomic,
         goto out;
     }
 
+    ucs_log_indent(+1);
     klm = UCT_IB_MLX5DV_ADDR_OF(create_mkey_in, in, klm_pas_mtt);
-    for (i = 0; i < ksm_data->mr_num; i++) {
+    ucs_carray_for_each(mr, ksm_data->mrs, ksm_data->mr_num) {
+        ucs_trace("klm[%ld] va %p mr [addr %p len %zu lkey 0x%x]",
```

Comment:
Seems like this trace is the same with one inside `uct_ib_mlx5_devx_reg_ksm_data_addr`. Maybe it worth to remove code duplication by asserts to separate functions (e.g. `uct_ib_mlx5_devx_klm_entry_log`)?